script.js

```diff
@@ -119,3 +124,38 @@ navAboutme.addEventListener('click', closeMenu);
 navFooter.addEventListener('click', closeMenu);
 burgerOpen.addEventListener('click', burgerMenu);
 burgerClose.addEventListener('click', burgerMenu);
+
+// Validation
+
+const email = document.getElementById('email');
+const contactForm = document.querySelector('#form');
+const footerSpan = document.querySelector('#footer-span');
+let emailValue = '';
+
+function validator(event) {
+  if (event.data !== null) {
+    if (event.data.toLowerCase() !== event.data) {
+      email.setCustomValidity('Please use LOWERCASE letters, Thanks!');
+      email.reportValidity();
+    } else if (email.validity.typeMismatch) {
+      email.setCustomValidity('please type a valid Email here!');
+      email.reportValidity();
+    } else {
+      email.setCustomValidity('');
+    }
+  } else if (event.data === null) footerSpan.style.display = 'none';
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md#:~:text=If%20the%20validation%20is%20not%20OK%2C%20you%20show%20an%20error%20message%20to%20the%20user%20near%20the%20submit%20button%20informing%20them%20of%20the%20error%20and%20the%20form%20is%20not%20sent.), Please make the errpr message appear near the submit button.

![image](https://user-images.githubusercontent.com/68796232/195426714-3dfcc2d4-3d6d-47ce-8271-309a68c8fe6f.png)

Reply:
Fixed.